themes/gokarna/layouts/partials/list.html

```diff
@@ -0,0 +1,14 @@
+<div class="container list-posts">
+
+    <h1 class="list-title">{{ .Name }}</h2>
```

Comment:
Made this `h1` since `h2` was looking small, please try change font-size/font-weight to fix this

Reply:
This was deliberately made as a `h2` later. We don't need a super big heading IMO

Reply:
Have you seen it as h2 vs h1 ? after adding the bold font, h2 seems smaller

Reply:
h1 looks good now